src/v2/BaseLoginRouter.js

```diff
@@ -179,6 +179,9 @@ export default Router.extend({
   },
 
   render: function(Controller, options = {}) {
+    if (sessionStorageHelper.getLastVisitedURL() !== window.location.href) {
```

Comment:
also add a comment on why widget needs to clear stateToken here.. 

"If url changes then widget assumes that user's intention was to initiate a new login flow hence remove clear stored token and use the newly bootstrapped token"